src/containers/temple/components/corusel/index.jsx

```diff
@@ -11,7 +11,7 @@ import styles from './carousel.scss';
 
 const cx = require('classnames');
```

Comment:
filename misspelled

Reply:
this component will be removed in the next iteration